### PR TITLE
[v0.4] bump rke to v1.5.11-rc4

### PR DESCRIPTION
**Pull request by kinarashah**
https://github.com/rancher/rancher/issues/45950

### CODE REVIEW

**Comment by kinarashah** [comment]
This is building fine locally for me with `dapper`, debugging why it's failing in CI.

**Comment by kinarashah** [comment]
glibc is present but 2.38 
``` 
docker run --rm registry.suse.com/bci/golang:1.21 /lib64/libc.so.6

GNU C Library (GNU libc) stable release version 2.38 (git 36f2487f13).
```

**Comment by kinarashah** [comment]
Needed `CGO_ENABLED=0` to make this work.

**Comment by tomleb** [comment]
> Needed CGO_ENABLED=0 to make this work.

There's a thread on Slack about it. There seems to be an issue with the upgrade of the bci golang image which interacts badly with the cache that's being kept in drone. Once [webhook release/v04 migrates to GHA](https://github.com/rancher/webhook/pull/384), you shouldn't have this problem anymore. We're working on getting this merged soon since we cannot release a new version of webhook with drone.

Once that's merged you should be able to rebase your PR, get rid of the CGO_ENABLED=0 change and have it work.

File: go.mod [hunk 1]
```diff
@@ -41,7 +41,7 @@ require (
 	github.com/rancher/dynamiclistener v0.4.0-rc2
 	github.com/rancher/lasso v0.0.0-20240123150939-7055397d6dfa
 	github.com/rancher/rancher/pkg/apis v0.0.0-20240618132447-2145c8e9d603
```
Comment (MbolotSuse) [inline go.mod:43]: Would you mind updating this commit to a more recent value as well?

**Review by MbolotSuse — COMMENTED** [review]
One comment on the rancher/rancher/pkg/apis version.

File: go.mod (hunk 1 above)
Comment (kinarashah) [reply go.mod]: Updated

**Comment by kinarashah** [comment]
Closing in lieu of https://github.com/rancher/webhook/pull/443. Not sure why but CI passed with fresh start on that PR.

**Pull request closed without merging**